libs/design-system/src/lib/Components/ToolsAndConcepts/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { TabMenu } from '../';
+import { ToolCard, ToolCardProps } from '../ToolCard';
+import { ButtonLink } from '../Button';
+import { ReactComponent as RightChevronIcon } from '../../../../images/arrows/chevron-right.svg';
+import { ReactComponent as ExternalLinkIcon } from '../../../../images/content/external-link.svg';
+
+export type ToolsAndConceptsProps = {
+  tools: ToolCardProps[];
+};
+
+const ToolsAndConcepts = ({ tools }) => (
+  <>
+    <div className="mb-2 text-h2">Tools and Concepts</div>
+    <p>Core concepts and tools you’ll need to get started building on Flow</p>
+    <TabMenu tabs={['Tools', 'Concepts']} onTabChange={() => null} />
+    <div className="grid gap-6 mt-9 md:grid-cols-2 md:gap-x-9 md:gap-y-6">
+      {tools.map((tool) => (
+        <ToolCard {...tool} />
+      ))}
+    </div>
+    <div className="flex flex-col justify-between mt-9 md:flex-row">
+      <ButtonLink
```

Comment:
I updated the ButtonLink component to include external and chevron icons: https://github.com/onflow/next-docs-v1/pull/111/files#diff-74f71e61cdec7349c426ca754f9f12c3b91494f59e68f65fab35e2050a443964R16